tokens.xml

```diff
@@ -7769,6 +7878,21 @@ Sacrifice this creature: This creature deals 1 damage to target creature or play
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Vampire     </name>
+            <text>Trample, Lifelink, Haste</text>
```

Comment:
Only the first keyword should be capitalised, to matched the printed token text.